pkg/protecode/protecode.go

```diff
@@ -103,11 +103,12 @@ func makeProtecode(opts Options) Protecode {
 
 //Options struct which can be used to configure the Protecode struct
 type Options struct {
-	ServerURL string
-	Duration  time.Duration
-	Username  string
-	Password  string
-	Logger    *logrus.Entry
+	ServerURL  string
+	Duration   time.Duration
+	Username   string
+	Password   string
+	UserApiKey string
```

Comment:
Tiny thing but could you please apply the naming convention `UserAPIKey`.

Reply:
Sure, thanks!